source/index.html.md

```diff
@@ -234,6 +234,143 @@ See the [export template format reference](./export_report_template.html) for mo
+| endDate | String | yyyy-mm-dd formatted date | The end date for which we will include expenses in the generated report.
+| domain | String | A valid domain, e.g.  "example.com"| Specifies the domain for which the reconciliation report will be run. **Note:** Only credentials generated for accounts that are domain admins on the provided domain will work. 
+| type | String |  "Unreported","All" | Specifies whether to run the report for only unreported card expenses, or for all card expenses.
+| async | Boolean | `true`, `false` | Specifies whether or not to run this command synchronously or asynchronously. **Note:** For the moment, only synchronous commands (`false`) are supported.
+**Optional elements** |
+| feed | String | Any name for a card feed, or `export_all_feeds` | Only the cards matching the specified feed will be exported.
+
+
+- `outputSettings` (Optional)
+
+Name | Format | Valid values | Description
+--------- | --------- | --------- | ---------
+| fileExtension | String | One of "csv", "txt", "json", "xml" | Specifies the format of the generated report. "csv" is the default.
+
+
+- `onFinish` (Optional)
+
+This section can be used to describe actions that need to be performed once the reconciliation report is generated.
+
+Action Name  | Description
+-------- | ---------
+email | Send an email linking to the generated report to a provided list of recipients.
+
+Name | Format | Valid values | Description
+-------- | ---------- | ---------------- | ---------
+**`email`** action | recipients | String | Comma-separated list of valid email addresses | People to email once the report is generated. The email will contain a link to download the report. |
+
+### Template parameter
+
+> - Template sample. See the [export template format reference](./export_report_template.html) for more information about how to write export templates.
+
+```
+<#-- Header Line -->
+Original Merchant,<#t>
+Posted date,<#t>
+Sales date,<#t>
+Modified Sales date,<#t>
+Original Amount,<#t>
+Modified Amount,<#t>
+<#list cards as card, reports>
+  <#list reports as report>
+      <#list report.transactionList as transaction>
+            ${transaction.originalMerchant},<#t>
+            ${transaction.posted},<#t>
+            ${transaction.originalCreated},<#t>
+            ${transaction.modifiedCreated},<#t>
+            ${(-transaction.originalAmount/100)?string("0.00")},<#t>
+            ${transaction.amountModified?then((-transaction.originalAmount/100)?string("0.00"), "")},<#t>
+        </#list>
+    </#list>
+</#list>
+```
+> - **Note:** Reconciliation templates follow a slightly different format than Export Templates. The basic iterative skeleton of the template is:
+
+```
+<#list cards as card, reports>
+    <#-- Properties to print per card -->
+    <#list reports as report>
+        <#-- Properties to print per report -->
+        <#list report.transactionList as transaction>
```

Comment:
we should use `as expense` for consistency, that's how we called the expense in all the other templates from the start.